package.json

```diff
@@ -28,6 +28,7 @@
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@mapbox/mapbox-gl-geocoder": "^5.0.1",
+    "@material-ui/core": "^4.12.4",
```

Comment:
`@material-ui/core` is part of material ui 4, we recently moved to 5 so components from core should be imported from `@mui/material`

Reply:
removed